### DISCUSSION TITLE
Is it possible to listen for count changes?

### DISCUSSION THREAD

**Discussion by ykaito21** [supabase/supabase#11259] [Questions]
Hi I found querying with count option. Is it possible to listen for count changes?

```
const { count, error } = await supabase
  .from('countries')
  .select('*', { count: 'exact', head: true })
```

**Comment by GaryAustin1** [comment]
You could use realtime to monitor inserts/deletes and would know the count changed.

**Reply by ykaito21** [reply]
So you mean if I want to know the count and the count change, I need the first and second step.
Is there any way to achieve this without using the first step?

**First Step:**
```
const { count, error } = await supabase
  .from('countries')
  .select('*', { count: 'exact', head: true })
```
**Second Step:**
```

supabase
  .channel('public:countries')
  .on('postgres_changes', { event: 'INSERT', schema: 'public', table: 'countries' }, handleRecordInserted)
  .on('postgres_changes', { event: 'DELETE', schema: 'public', table: 'countries' }, handleRecordDeleted)
  .subscribe()
```

**Comment by sunilguptasg** [comment]
Perhaps you can have a trigger that in turn updates another table with the count, and you could then subscribe to that table...

**Reply by ykaito21** [reply]
Thanks! If I want to get the count and the count change, that maybe a good option for now.
